### DISCUSSION TITLE
About the pycortex db

### DISCUSSION THREAD

**Discussion by geoffreyfan** [kendrickkay/nsdcode#25] [Q&A] [answered]
Hello, could someone please share the pycortex db file for the participants in the NSD dataset with me? Thank you very much!

**Comment by kendrickkay** [comment, marked as answer]
Hello, I'm not sure this is the appropriate place for this question (NSD does not come with any pycortex-specific files).

**Reply by geoffreyfan** [reply]
![image](https://github.com/cvnlab/nsdcode/assets/105597268/d7c3576f-8d48-47d5-802a-9faa3aa969f2)
Hello! I noticed that you're discussing sharing pycortex db files. May I ask where I can download them? Thank you very much!
